plugin/src/main/resources/plugin.yml

```diff
@@ -1,6 +1,6 @@
- name: Tabbed
- version: 1.7
- description: This plugin provides the "Tabbed" library which is necessary for some plugins.
- authors: [Keenan Thompson]
- website: https://github.com/thekeenant/tabbed
- main: com.keenant.tabbed.TabbedPlugin
\ No newline at end of file
+name: Tabbed
```

Comment:
please exclude changes to this file

Reply:
it's no big deal it just removed extra spaces this should be fine